content/tutorials/Team5.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Servos with ESP32 DevBoard
+date: 2024-12-06
+authors:
+  - name: Andrew Lara
```

Comment:
Nobody else? Your teammates (which are required to help) did not? Are you sure?

Reply:
Changes have been committed!

Reply:
Okey dokey!